modules/solid_mechanics/doc/content/source/materials/ComputeMultipleInelasticCosseratStress.md

```diff
@@ -1 +1,27 @@
-!template load file=stubs/moose_object.md.template name=ComputeMultipleInelasticCosseratStress syntax=/Materials/ComputeMultipleInelasticCosseratStress
\ No newline at end of file
+# ComputeMultipleInelasticCosseratStress
+
+!syntax description /Materials/ComputeMultipleInelasticCosseratStress
+
+The `ComputeMultipleInelasticCosseratStress` performs the same functions as the [ComputeMultipleInelasticStress.md]
+material, but additionally computes:
+
+- the coupled Cosserat elastic stress $\boldsymbol{\sigma}_c$
+- the derivative tensor of the coupled Cosserat elastic stress with regards to strain
+
+These two terms are computed in the same way as [ComputeCosseratLinearElasticStress.md].
+If finite strain rotations are considered, then the these two terms are further modified with the rotation increment $R$:
+
+!equation
+$\boldsymbol{\sigma}_c$ := R $\boldsymbol{\sigma}_c$ R^T
```

Comment:
This equation is not correctly rendered. It might be the $ symbols.

Reply:
fixed